downloaders/dumps_downloader.py

```diff
@@ -0,0 +1,131 @@
+"""
+Download Github issues daily dumps from this page
+"http://ghtorrent-downloads.ewi.tudelft.nl/mongo-daily/" and store them into target directory.
+"""
+import tarfile
+import os
+import argparse
+from typing import Optional
+
+import urllib.request
+from bs4 import BeautifulSoup
+from tqdm import tqdm
+
+
+def download_progress_hook(p_bar: tqdm) -> (int, int, Optional[int]):
```

Comment:
return type looks strange - probably you missed [callable](https://docs.python.org/3/library/typing.html#typing.Callable) - and it should look like 
```suggestion
def download_progress_hook(p_bar: tqdm) -> Callable[int, int, Optional[int]]:
```